src/OpenSage.Game/Game.cs

```diff
@@ -676,10 +676,10 @@ public AptWindow LoadAptWindow(string aptFileName)
                         players[i].SelectUnits(new[] { startingBuilding });
                     }
 
-                    if (players[i].IsHuman)
-                    {
-                        localPlayerIndex = i;
-                    }
+                    //if (players[i].IsHuman)
```

Comment:
This would overwrite the localPlayerIndex passed to  `StartGame`. Not sure why it was in there

Reply:
Maybe the point was to autoselect the right player in single player maps?

Regardless, if this code is no longer deemed necessary it should be deleted.